tool/options.go

```diff
@@ -60,12 +60,9 @@ func ProcessFlags() {
 		"Configuration file pathname to managed kubernetes cluster",
 	)
 
-	flag.BoolVar(
-		&Options.EnableLease,
-		"enable-lease",
-		false,
-		"If enabled, the controller will start the lease controller to report its status",
-	)
+	flag.BoolVar(&Options.EnableLeaderElection, "leader-elect", true,
+		"Enable leader election for controller manager. "+
+			"Enabling this will ensure there is only one active controller manager.")
```

Comment:
Does this runtime change need to also be done in the grc chart? Or something with the addon framework?